src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_ITypeOfExpression.vb

```diff
@@ -9,7 +9,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests.Semantics
     Partial Public Class IOperationTests
         Inherits SemanticModelTestBase
 
-        <CompilerTrait(CompilerFeature.IOperation)>
+        <CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)>
```

Comment:
This isn't a dataflow test, I assume you meant to add this to the newly added test below.